src/adsp_config.c

```diff
@@ -1532,6 +1532,33 @@ static int parse_adsp_config_v2_5(const toml_table_t *toml, struct adsp *out,
 	return 0;
 }
 
+static int parse_version(toml_table_t *toml, int64_t version[2])
```

Comment:
What constitutes a correct version string here, can you say in an inline comment.

Reply:
version is not stored in string, it is in toml array of integers. I've added this information as comment.